data/talks.yml

```diff
@@ -1,4 +1,22 @@
 -
+  date: '2017-04-06'
+  talks:
+    -
+      title: Deploy Rails with Docker (For Dummies)
+      url: http://ruby-nord-kontena.surge.sh/#/
+      level: confirmed
```

Comment:
Is it really a `confirmed` level? 
There's "For Dummies" in the title 😬 

Same for the other talks, are they all confirmed?